geocode.go

```diff
@@ -14,13 +14,17 @@ import (
 type GoogleGeo struct {
 	client *http.Client
 	apiKey string
+	region string
 }
 
 // NewGoogleGeo returns a new GoogleGeo instance
-func NewGoogleGeo(apiKey string) *GoogleGeo {
+// The region parameter is optional, read about it here:
+// https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes
+func NewGoogleGeo(apiKey, region string) *GoogleGeo {
```

Comment:
Sure, either that or `func (g *GoogleGeo) SetRegion(region string)`. What do you prefer?